source/index.js

```diff
@@ -84,7 +84,7 @@ ConnectionManager.prototype.__loadDialectModule = ConnectionManager.prototype._l
 ConnectionManager.prototype._loadDialectModule = function (...args) {
   const pg = this.__loadDialectModule(...args);
   pg.types.setTypeParser(20, function (val) {
-    if (val > Number.MAX_SAFE_INTEGER) return BigInt(val);
+    if (val > Number.MAX_SAFE_INTEGER) return String(val);
```

Comment:
I just found out about this issue, sorry for catching this late:
https://github.com/sequelize/sequelize/issues/10468

It seems like from a ux perspective, sometimes users do expect BigInt and not string. I think we should aim to keep this as BigInt.

Reply:
According to our last meeting, we've decided to trust in Sequelize's core. We'll keep treating those as Strings for now.